.github/workflows/test.yaml

```diff
@@ -145,7 +145,8 @@ jobs:
         run: |
           mkdir -p local-deployment
           cd local-deployment
-          qhub init local --project=thisisatest  --domain github-actions.qhub.dev --auth-provider=password --terraform-state=local
+          qhub init local --project=thisisatest  --domain jupyter.github-actions.qhub.dev --auth-provider=password --terraform-state=local
```

Comment:
Do we need `jupyter.` in here?

Reply:
No we don't I only did this for consistency. Should we just use `github-actions.qub.dev` so as to set the example that `jupyter` is no longer required?

Reply:
I think that will be great.